app/views/deploys/active.erb

```diff
@@ -12,6 +12,6 @@
     <%= render 'job_executions', job_executions: active %>
 
     <h2>Queued</h2>
-    <%= render 'job_executions', job_executions: queued %>
+    <%= render 'job_executions', job_executions: JobQueue.grouped_queue %>
```

Comment:
this looks suspicious ... should this assign `queue = JobQueue.grouped_queue` somewhere ?

Reply:
What do you think about debug accepting a `grouped` argument (default to false) and handle the grouping in `debug`. Then we can call it here by passing true?

Reply:
how about debug keeps returning what it returned before: a grouped queue ... then less tests have to change ?